src/loaders/helmet.js

```diff
@@ -0,0 +1,7 @@
+const express = require("./express").expressLibrary;
+
+const helmet = require("helmet");
+
+const app = express();
```

Comment:
We shouldn't create a new instance of express app, we are supposed to use the same one that it's used in app.js and loaders/index.js
Try by importing app from app.js, not sure if it'll work, if it doesn't then you will need to transform this module to export a function which will receive app as a param
```
function helmetLoader(app) {
     app.use(helmet());
}
```
and then in loaders/index.js
```
helmetLoader(app);
```